.concourse/tasks/upgrade.sh

```diff
@@ -35,13 +38,13 @@ gcloud --quiet beta container \
   --disk-type "pd-ssd" \
   --disk-size "100" \
   --metadata disable-legacy-endpoints=true \
-  --scopes "https://www.googleapis.com/auth/devstorage.read_only","https://www.googleapis.com/auth/logging.write","https://www.googleapis.com/auth/monitoring","https://www.googleapis.com/auth/servicecontrol","https://www.googleapis.com/auth/service.management.readonly","https://www.googleapis.com/auth/trace.append" \
+  --scopes "https://www.googleapis.com/auth/devstorage.read_only,https://www.googleapis.com/auth/logging.write,https://www.googleapis.com/auth/monitoring,https://www.googleapis.com/auth/servicecontrol,https://www.googleapis.com/auth/service.management.readonly,https://www.googleapis.com/auth/trace.append" \
   --preemptible \
   --num-nodes "1" \
   --enable-stackdriver-kubernetes \
   --enable-ip-alias \
   --network "projects/${GKE_PROJECT}/global/networks/default" \
-  --subnetwork "projects/${GKE_PROJECT}/regions/$(echo ${GKE_ZONE} | sed 's/-.$//')/subnetworks/default" \
+  --subnetwork "projects/${GKE_PROJECT}/regions/${GKE_ZONE//-.$/}/subnetworks/default" \
```

Comment:
This is wrong; it was changed from a regular expression to (essentially) a shell glob, in which case `$` lost its meaning as the end-of-line matcher.
```suggestion
  --subnetwork "projects/${GKE_PROJECT}/regions/${GKE_ZONE%-?}/subnetworks/default" \
```
```bash
$ GKE_ZONE=hello-a
$ echo "${GKE_ZONE%-?}"
hello
```